be/src/exec/pipeline/pipeline_builder.h

```diff
@@ -32,7 +32,7 @@ class PipelineBuilderContext {
             : _fragment_context(fragment_context), _degree_of_parallelism(degree_of_parallelism) {}
 
     void add_pipeline(const OpFactories& operators) {
-        _pipelines.emplace_back(std::make_unique<Pipeline>(next_pipe_id(), operators));
+        _pipelines.emplace_back(std::make_shared<Pipeline>(next_pipe_id(), operators));
```

Comment:
why?

Reply:
Because the type of `_pipelines` is `std::vector<std::shared_ptr<Pipeline>>`, this is a typo.